htdocs/resources/views/master_layout.blade.php

```diff
@@ -11,8 +11,8 @@
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
     <title>@yield('title')</title>
 
-    <!-- Bootstrap -->
-    <link href="http://{{ $server_host }}/css/bootstrap.min.css" rel="stylesheet">
+    <!-- Load All CSS from a single concatenated file. See resources/assests/less/app.less  -->
+    <link rel="stylesheet" href="{{ elixir("css/app.css") }}">
```

Comment:
all css is bundled in this one. This file is versioned so it is cash busted every time I build a new one